src/dsp_tools/commands/excel2json/resources.py

```diff
@@ -197,47 +191,50 @@ def excel2resources(
             and the success status (True if everything went well)
     """
 
-    # load file
-    all_classes_df = read_and_clean_excel_file(excelfile=excelfile)
-    all_classes_df = prepare_dataframe(
-        df=all_classes_df,
+    df_dict = read_and_clean_all_sheets_excelfile(excelfile)
+    classes_df = df_dict["classes"]
+    df_dict.pop("classes")
```

Comment:
```suggestion
    classes_df = df_dict.pop("classes")
```

I may be wrong, but I think that should work?